coffee_core/src/coffee.rs

```diff
@@ -442,8 +448,70 @@ impl PluginManager for CoffeeManager {
         Err(err)
     }
 
-    async fn nurse(&mut self) -> Result<(), CoffeeError> {
-        self.recovery_strategies.scan().await
+    async fn nurse(&mut self) -> Result<CoffeeNurse, CoffeeError> {
+        let status = self.recovery_strategies.scan(self).await?;
+        let mut nurse_actions: Vec<NurseStatus> = vec![];
+        for defect in status.defects.iter() {
+            log::debug!("defect: {:?}", defect);
+            match defect {
+                Defect::RepositoryLocallyAbsent(repos) => {
```

Comment:
> are you checking the storage exist but the repositories directory no?

Yes
It's confusing now because it's the only case 
later we will also have `RepositoryAbsentInConfiguration`  which is the case when a repo exists on disk but not in `coffee` configuration